superset-frontend/package.json

```diff
@@ -160,7 +160,7 @@
     "react-split": "^2.0.9",
     "react-sticky": "^6.0.3",
     "react-syntax-highlighter": "^15.3.0",
-    "react-table": "^7.2.1",
+    "react-table": "^7.6.3",
```

Comment:
Bumping to be in line with the version used in the new version of table chart: https://github.com/apache-superset/superset-ui/pull/1034